django/utils/_os.py

```diff
@@ -56,18 +56,17 @@ def symlinks_supported():
     host platform and/or if they are allowed to be created (e.g.
     on Windows it requires admin permissions).
     """
-    tmpdir = tempfile.mkdtemp()
-    original_path = os.path.join(tmpdir, 'original')
-    symlink_path = os.path.join(tmpdir, 'symlink')
-    os.makedirs(original_path)
-    try:
-        os.symlink(original_path, symlink_path)
-        supported = True
-    except (OSError, NotImplementedError, AttributeError):
-        supported = False
-    else:
-        os.remove(symlink_path)
-    finally:
-        os.rmdir(original_path)
-        os.rmdir(tmpdir)
-        return supported
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
It would be nice to be consistent: either `temp_dir` or `tmpdir` (although if an existing test is using some other form and is otherwise unmodified, I wouldn't change it).